code/addons/toolbars/src/components/ToolbarMenuList.tsx

```diff
@@ -18,11 +18,12 @@ export const ToolbarMenuList: FC<ToolbarMenuListProps> = withKeyboardCycle(
     description,
     toolbar: { icon: _icon, items, title: _title, preventDynamicIcon, dynamicTitle },
   }) => {
-    const [globals, updateGlobals] = useGlobals();
+    const [globals, updateGlobals, storyGlobals] = useGlobals();
```

Comment:
This is the API that addons use to tell if the user has overridden a global value they are interested in.

If `storyGlobals[X]` is set, then the value of `globals.X` is hard-coded (for now) and so showing a read/write UI is probably not sensible.